### PR TITLE
Add info about where Minio mounts its filesystem

### PR DESCRIPTION
**Pull request by billmei**
Fixes https://github.com/minio/minio/issues/5341 by adding clarifying documentation information.

See the issue for more details.

I'm not sure if this is the right location for this documentation, but am happy to move it to a different file if there is a more appropriate place to put this.

## Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/browser/README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ Currently these tokens expire after 10hrs, this is not configurable yet.
 minio server /data
 ```
 
+This will mount the filesystem that Minio uses in the `/data` directory of the host machine. For example, if you run Minio locally and create a bucket called `foo`, then you can access any files you "uploaded" to the `foo` bucket by navigating to the `/data/foo` directory on your local disk.
```
Comment (harshavardhana) [inline docs/browser/README.md:18]: Minio doesn't mount anything, minio simply uses what is already present on the disk. Minio has other modes of operation as well so the line indicating accessing backend is not correct. In-fact it is discouraged to access the backend i.e /data directly even if its possible. 

Minio server manages concurrent access to backends - accessing backend directly would defeat this and lead to data loss and application issues.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: docs/browser/README.md (hunk 1 above)
Comment (billmei) [reply docs/browser/README.md:18]: Interesting... My use case is to use minio as a scriptable front-end interface to my local filesystem, as I'm building an app on top of minio an alternative to using Finder.

Comment (harshavardhana) [reply docs/browser/README.md:18]: You can @billmei but this doc change might not be needed IMO.

Comment (billmei) [reply docs/browser/README.md:18]: Ok, I will close this pull request.

**Pull request closed without merging**